addons/purchase_stock/models/product.py

```diff
@@ -22,6 +22,13 @@ class ProductTemplate(models.Model):
         'account.account', string="Price Difference Account", company_dependent=True,
         help="This account is used in automated inventory valuation to "\
              "record the price difference between a purchase order and its related vendor bill when validating this vendor bill.")
+    first_seller_id = fields.Many2one('res.partner', compute='_compute_first_seller_id', store=True)
```

Comment:
You still need all those fields? They seem to not be use

Reply:
This field is used to get the first supplier of the product to group by in the "Reordering Rules" list view.

Reply:
shouldn't be a default filter?

Reply:
And we could use a compute directly on orderpoint rahter than doing a computed field here with a related M2o

Reply:
Maybe we should just add the field on orderpoint since it's used only there.